b2g_util/util/adb_helper.py

```diff
@@ -101,6 +101,51 @@ def adb_push(cls, source, dest, serial=None):
         return output
 
     @classmethod
+    def adb_forward(cls, command=None, source=None, dest=None, serial=None):
+        """
+        Use port forwarding on device.
+        @return: the stdout and return code (from device) of "adb forward" command. e.g. (stdout, retcode)
+        @raise exception: When return code (from adb command) isn't zero.
+        When using remove command, only dest should be mentioned, e.g. adb_forward('remove', dest="tcp:12345")
+        """
+        if serial is None:
+            cmd = 'adb forward'
+        else:
+            cmd = 'adb -s %s forward' % (serial,)
+        # get returncode from device
+        if command:
+            cmd += " --%s" % (command,)
+            if 'remove' == command:
```

Comment:
`adb forward --no-rebind <local> <remote>` will fail here.
I am trying to add the commit for it.